invenio_jobs/assets/semantic-ui/js/invenio_jobs/administration/search/JobRunsHeader.js

```diff
@@ -0,0 +1,75 @@
+// This file is part of Invenio
+// Copyright (C) 2024 CERN.
+//
+// Invenio RDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import { RunButton } from "./RunButton";
+import React, { Component } from "react";
+import { NotificationContext } from "@js/invenio_administration";
+import { i18next } from "@translations/invenio_app_rdm/i18next";
+import PropTypes from "prop-types";
+import { http } from "react-invenio-forms";
+
+export class JobRunsHeaderComponent extends Component {
+  constructor(props) {
+    super();
+
+    this.state = {
+      jobId: props.jobId,
+      title: "Job Details",
+      description: "",
+      config: {},
+    };
+  }
+
+  componentDidMount() {
+    const { jobId } = this.state;
+    http
```

Comment:
This should be a cancellable Promise, see [example](https://github.com/inveniosoftware/invenio-app-rdm/blob/6a731fad99161bd49596799eacbce13517905162/invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/frontpage/RecordsList.js#L48)

Reply:
Correct me if I'm wrong @zzacharo, but I think that in this case where there is only one call per component, the one use case to use a cancellable Promise is to cancel the call in `componentWillUnmount`, in order to avoid having a zombie network request trying to update an unmounted component.